### PR TITLE
Show details of protocol startup

### PR DESCRIPTION
**Pull request by clmckinley**

### CODE REVIEW

**Comment by MatthewGerber** [comment]
@rd3fa @clmckinley Is this finished?

**Comment by rd3fa** [comment]
@MatthewGerber  yes as of this morning we have finished this.

File: Sensus.Shared/UI/ProtocolsPage.cs [hunk 1]
```diff
@@ -311,55 +414,55 @@ public ProtocolsPage()
...
-                                new InputGroup[] { new InputGroup { Name = "Please Wait", Inputs = { new LabelOnlyInput("Retrieving participation information.", false) } } },
-                                cancellationTokenSource.Token,
-                                false,
-                                "Cancel",
-                                null,
-                                null,
-                                null,
-                                false,
-                                async () =>
+                            null,
```
Comment (MatthewGerber) [inline Sensus.Shared/UI/ProtocolsPage.cs]: @clmckinley @rd3fa Were any changes made to the following lines? If not, please revert formatting.

File: Sensus.iOS/Sensus.iOS.csproj [hunk 2]
```diff
@@ -45,8 +45,7 @@ Store. It is possible to run and debug this project on physical and virtual iOS
     <WarningLevel>4</WarningLevel>
```
Comment (MatthewGerber) [inline Sensus.iOS/Sensus.iOS.csproj]: Please revert this file.

**Pull request merged**